etc/sysconfig/pacemaker.in

```diff
@@ -212,6 +212,15 @@
 #
 # Default: PCMK_remote_port="3121"
 
+# PCMK_remote_pid1 (Advanced Use Only)
+#
+# Specify behavior when the Pacemaker Remote process is not running as PID 1.
```

Comment:
I'm thinking:

When a bundle resource's "run-command" option is left to default, Pacemaker Remote runs as PID 1 in the bundle's containers. When it does so, it loads environment variables from the container's /etc/pacemaker/pcmk-init.env, and performs the PID 1 responsibility of reaping dead subprocesses. This variable controls whether those behaviors are performed when Pacemaker Remote is not running as PID 1. It is intended primarily for developer testing but can be useful when "run-command" is set to a separate, custom PID 1 process that launches Pacemaker Remote.